models/comments_model.js

```diff
@@ -66,4 +65,47 @@ exports.insertComments = ( id, commentbody) => {
                 return result.rows[0];
               })
     
-      }
\ No newline at end of file
+      };
+      exports.removeCommentById = (id) =>{
+        const { comment_id } = id;
+        const psqlDeleteQuery = ` 
+        DELETE FROM comments 
+        WHERE comment_id = $1; `
+       
+        const firstPsqlQuery = `SELECT * FROM comments WHERE comment_id = $1;`
+        return db.query(firstPsqlQuery,[comment_id] )
+        .then((results) => {
+            if (results.rows.length === 0){
+                return Promise.reject({ status: 404, msg: "Not found" });
+            } else { return db.query(psqlDeleteQuery, [comment_id])
+                .then((results) =>{
+    
+                    return results.rows})
+                }
```

Comment:
You;ve nested .thens. 

i think you've also made the check comments functions, but then not used it anywhere